app/uk/gov/hmrc/leakdetection/scanner/ExemptionChecker.scala

```diff
@@ -30,9 +30,17 @@ class ExemptionChecker @Inject()() {
       .filterNot(exemption =>
         excludedResults.exists(exclusion =>
           exemption.ruleId == exclusion.ruleId &&
-            exemption.filePath == exclusion.filePath &&
+            removeLeadingSlash(exemption.filePath) == removeLeadingSlash(exclusion.filePath) &&
             exemption.text.fold(true)(exclusion.lineText.contains)
         )
       )
   }
-}
+
+  private def removeLeadingSlash(filePath: String) = {
+    filePath match {
+      case null => filePath
```

Comment:
why do we have `null`? Can we prevent during the parse (or return `""` rather than preserve the null)

Reply:
Used a custom json formatter to ensure non-null, though it looks impossible to have null in the normal code flow.